gradle/generation/javacc.gradle

```diff
@@ -129,17 +115,33 @@ configure(project(":solr:core")) {
 
 // We always regenerate, no need to declare outputs.
 class JavaCCTask extends DefaultTask {
-  @Input
+  @InputFile
   File javaccFile
```

Comment:
should this be `RegularFile`?

Reply:
It doesn't have to be. It can be plain Java API. We're not being fancy with the lazy API here so I don't think we need to make things complicated? There is a semantic difference between Input and InputFile - this was a bug.

https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:up_to_date_checks